src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/DlqMessagesToDatabaseTest.java

```diff
@@ -44,18 +57,17 @@ public void should_store_dlq_messages_in_database() {
         );
 
         await().ignoreException(AssertionError.class)
-            .pollInterval(500, MILLISECONDS)
-            .atMost(30, SECONDS)
+            .pollInterval(2, SECONDS)
```

Comment:
Should make these values constants so that if we ever need to change them again, it can be done in one place